src/libraries/System.Private.CoreLib/src/System/Numerics/BitOperations.cs

```diff
@@ -708,5 +709,195 @@ public static nuint RotateRight(nuint value, int offset)
             return (nuint)RotateRight((uint)value, offset);
 #endif
         }
+
+        private static class Crc32Fallback
+        {
+
+            private static readonly uint[] s_crcTable = Crc32ReflectedTable.Generate(0x82F63B78u);
+
+            internal static uint Crc32C(uint crc, byte data)
+            {
+                if (!BitConverter.IsLittleEndian)
+                {
+                    data = BinaryPrimitives.ReverseEndianness(data);
```

Comment:
No harm in it, but this isn't necessary for the `byte` overload, as you can't swap the order of a single byte.  As the docs say:
> This method effectively does nothing and was added only for consistency.